ChangeLog

```diff
@@ -62,6 +62,11 @@ Release date: TBA
 
   Closes #5460
 
+* Disable checker ``bad-docstring-quotes`` for Python <= 3.7, because in this versions the line
+  numbers decorated functions and classes are not reliable
+
+  Closes #3077
```

Comment:
```suggestion
* Disable checker ``bad-docstring-quotes`` for Python <= 3.7, because in these versions the line
  numbers for decorated functions and classes are not reliable which interferes with the checker.

  Closes #3077
```